app/iht/utils/CommonHelper.scala

```diff
@@ -502,4 +502,6 @@ object CommonHelper {
       url
     }
   }
+
+  def getNinoFromSession(request:Request[_]): String = request.session.get(Constants.NINO).fold("")(identity)
```

Comment:
Would this not be better if it returned an Option? An unfound Nino could then be returned as a None.